build.gradle

```diff
@@ -291,4 +293,33 @@ project(':vgen') {
       args = clargs.split(',').toList()
     }
   }
+
+  task verifyLicense << {
+    def licenseText = '''/* Copyright 2016 Google Inc
```

Comment:
I uploaded a change to move the license into its own file. It feels a little funny to have it in the top-level, but I'm not sure where else to put it. Let me know what you think.